docs/cep/query-guide/functions/time/dateDiff.md

```diff
@@ -51,7 +51,7 @@ This query calculates the date difference between the given timestamp values '14
 
 ```sql
 CREATE STREAM InputStream (startDate string, endDate string, inputFormat string);
-CREATE STREAM OutputStream (dateDifference int);
+CREATE SINK STREAM OutputStream (dateDifference int);
```

Comment:
@StoyanTc @diana-macrometa time:dateDiff seems to be returning values in "long" format instead of "int" .. Either documentation needs to be updated, or is it a bug?.. easiest way though is to update document ..

<INT> time:dateDiff(<STRING> date.value1, <STRING> date.value2, <STRING> date.format1, <STRING> date.format2)
<INT> time:dateDiff(<STRING> date.value1, <STRING> date.value2)
<INT> time:dateDiff(<LONG> timestamp.in.milliseconds1, <LONG> timestamp.in.milliseconds2)

TO:

<LONG> time:dateDiff(<STRING> date.value1, <STRING> date.value2, <STRING> date.format1, <STRING> date.format2)
<LONG> time:dateDiff(<STRING> date.value1, <STRING> date.value2)
<LONG> time:dateDiff(<LONG> timestamp.in.milliseconds1, <LONG> timestamp.in.milliseconds2)

And change the example to:
CREATE SINK STREAM OutputStream (dateDifference long);

Reply:
@StoyanTc, should I update the documentation, or do we want to file a bug?

Reply:
@diana-macrometa - I agree with @koshy-thomas , let's fix the doc, the diff between 2 days is long.